sql/sql_alter.cc

```diff
@@ -386,10 +386,12 @@ bool Sql_cmd_alter_table::execute(THD *thd) {
     wsrep::key_array keys;
     // append tables referenced by this table
     // append tables that are referencing this table
-    if (wsrep_append_fk_parent_table(thd, first_table, &keys) ||
-        wsrep_append_child_tables(thd, first_table, &keys)) {
-      WSREP_DEBUG("TOI replication for ALTER failed");
-      return true;
+    for (Table_ref *table = first_table; table; table = table->next_global) {
+      if (wsrep_append_parent_tables(thd, table, &keys) ||
+          wsrep_append_child_tables(thd, table, &keys)) {
+        WSREP_DEBUG("TOI replication for ALTER failed");
```

Comment:
Optional: It would be nice to print the table name that ALTER failed on.

Reply:
Good point. Instead of doing it here, I have added the log inside `wsrep_append_parent_tables`/`wsrep_append_child_tables` functions